src/test/java/poongduck/board/controller/BoardControllerTest.java

```diff
@@ -32,69 +31,48 @@
 class BoardControllerTest{
 
 	@Autowired
-    MockMvc mockMvc;
+	MockMvc mockMvc;
 	
 	@Autowired
 	BoardService boardService;
 	
 	@Autowired
 	BoardRepository boardRepository;
 	
-	//@Disabled
-    @Test
-    @DisplayName("Board Controller 기본 테스트")
-    public void board() throws Exception {
+	@Test
+	@DisplayName("openBoardList 메소드 테스트")
+	public void test_openBoardList() throws Exception {
+		//should
     	List<BoardEntity> expectList = createBoardListFroTest();
     	
+    	//when
     	MvcResult result =mockMvc.perform(get("/board"))
-                .andExpect(status().isOk())
-                .andExpect(view().name("/board/boardList"))
-                .andExpect(model().attributeExists("list"))
-                .andDo(print())
-                .andReturn();
+    			.andExpect(status().isOk())
+    			.andExpect(view().name("/board/list"))
+    			.andExpect(model().attributeExists("list"))
+    			.andDo(print())
+    			.andReturn();
         
     	Object actualList = result.getModelAndView().getModel().get("list");
-         
-        assertReflectionEquals(expectList, actualList, ReflectionComparatorMode.IGNORE_DEFAULTS);
-    }
-    
-    //@Disabled
-    @Test
-    @DisplayName("Board Service 메소드 기본 테스트")
-    public void test_boardService_basic() throws Exception {
-    	
-    	List<BoardEntity> expectList = createBoardListFroTest();
-    	
-    	List<BoardEntity> actualList = boardService.selectBoardList();
-		
-		assertReflectionEquals(expectList, actualList, ReflectionComparatorMode.IGNORE_DEFAULTS);
-    }
-    
-    @Test
-    @DisplayName("BoardRepository 기본 테스트")
-    public void test_BoardRepository_basic() throws Exception {
-    	
-    	List<BoardEntity> expectList = createBoardListFroTest();
-    	
-    	List<BoardEntity> actualList = boardRepository.findAllByOrderByIdDesc();
-		
-		assertReflectionEquals(expectList, actualList, ReflectionComparatorMode.IGNORE_DEFAULTS);
+
+    	assertReflectionEquals(expectList, actualList, ReflectionComparatorMode.LENIENT_DATES);
     }
     
 	public List<BoardEntity> createBoardListFroTest() {
```

Comment:
```suggestion
	public List<BoardEntity> createBoardListForTest() {
```
오타수정~

Reply:
nitpick, 

개인적으론 
```suggestion
	public List<BoardEntity> createDummyBoardList() {
```
or
```suggestion
	public List<BoardEntity> createMockBoardList() {
```
정도로 하면 좋을듯~

Reply:
resolve conversation은 리뷰해주시는 사람이 대개는 닫도록 되어있고, 반영되면, 코멘트에 적용한 커밋 sha 값만 남겨주면 확인하고 내가 닫을게~

Reply:
c41df9e 에서 수정함:heart: 

--> 요런 식으로 남겨주면 리뷰어가 `Resolve conversation` 버튼을 눌러주도록 되어있어~

Reply:
이거는 무튼 수정된듯 하니 `Resolve conversation` 누르겠음 😄